hack/check-skaffold-deps.sh

```diff
@@ -18,76 +18,21 @@ set -e
 
 echo "> Check Skaffold Dependencies"
 
-check_successful=true
-
-out_dir=$(mktemp -d)
-function cleanup_output {
-  rm -rf "$out_dir"
-}
-trap cleanup_output EXIT
-
-function check() {
```

Comment:
Perhaps we could keep the check method. See below for a more detailed explanation.